packages/core/stories/colors/DefaultColorPalette.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
+const families = [
+  { color: 'blue-slate', role: 'primary', d: 'toolbar / sidebar' },
+  { color: 'digital-blue', role: 'secondary', d: 'accessible buttons / links' },
+  { color: 'pantone-red', role: 'error', d: 'dialogs, button states' },
+  { color: 'bright-yellow', role: 'warning / alert', d: 'warning messages, etc.' },
+  { color: 'cool-gray', role: 'UX Grays', d: 'bodies, lines, text' },
+  { color: 'pond-green', role: 'success', d: 'success buttons, etc.' },
+];
+
+const tintsAndShades = (color) => {
+  const vals = [
+    100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900,
+  ];
+  return vals.map((v) => ({
+    c: `${color}-${v}`,
+    v,
+  }));
+};
+
+</script>
+
+<style>
+
+.description {
+    padding-left: var(--space-base);
+    padding-right: var(--space-base);
+    font-style: italic;
+    font-weight: normal;
+    width: calc(var(--increment) * 3);
+    height: var(--increment);
+    font-size: .875rem;
+    color: var(--cool-gray-600);
+}
+
+.color.role {
+    font-weight: 900;
+    text-transform:uppercase;
+    color: var(--cool-gray-800);
+}
+
+.swatches {
+    display:flex;
+    flex-wrap: row-wrap;
+    font-size:.85rem;
+    margin-bottom: var(--space-8x);
+}
+
+.color {
+    height: var(--increment);
+    width: calc(var(--increment) * 3);
+    display: grid;
+    align-content: center;
+    padding-left: var(--space-base);
+    padding-right: var(--space-base);
+    font-weight: 100;
+}
+
+.dark {
+    color: white;
+}
+
+</style>
+
+<div class=story>
+    <h1 class="story__title">Color Palette for Telemetry</h1>
```

Comment:
Let's change this to
```suggestion
    <h1 class="story__title">Default Color Palette</h1>
```